runners/flink/build.gradle

```diff
@@ -87,7 +87,7 @@ dependencies {
   validatesRunner project(path: project.path, configuration: "shadow")
   //LYFT CUSTOM
   shadow "org.apache.flink:flink-connector-kafka-0.11_2.11:1.5-lyft20180920"
-  shadow "com.lyft:streamingplatform-kinesis:1.2.20181003"
+  shadow "com.lyft:streamingplatform-kinesis:1.2-SNAPSHOT"
```

Comment:
Needs to be SNAPSHOT until a new release of streamingplatform containing https://github.com/lyft/streamingplatform/pull/414 is released

Reply:
Verified that we have the matching snapshot in the repo.